sequencer/src/options.rs

```diff
@@ -197,8 +201,12 @@ enum SequencerModule {
     /// * query: add query service endpoints
     /// * submit: add transaction submission endpoints
     Http(Module<api::options::Http>),
-    /// Alias for query-fs.
-    Query(Module<api::options::Fs>),
+    /// Alias for query-sql.
```

Comment:
Should we give @Ancient123's a heads up that the default query service backend will change to postgres after this is merged?

Reply:
Not just me. But also other people.

Reply:
Yes. It's easy enough to just change `query` to `query-fs` to get the old behavior. But we should switch our infra over to Postgres soon